### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by Georgeoak03**
End of day 12/7
Semantics updated
Responsiveness improved

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -134,10 +135,11 @@ a {
...
 
+
 .project-1, .project-2, .project-3 {
 	display: flex;
 	flex-wrap: wrap;
-	padding-top: 50px;
+	padding-top: 100px;
 }
 
 .projecttitle1, .projecttitle2, .projecttitle3 {
```
Comment (Harry-Johnston) [inline portfolio.css:145]: maybe add a hyphen or underscore between names to make them easier to read

File: portfolio.css [hunk 2]
```diff
@@ -153,7 +155,7 @@ a {
 	display: flex;
```
Comment (Harry-Johnston) [inline portfolio.css:155]: separate the names of projectmaintext with hyphens or underscores

**Review by Harry-Johnston — CHANGES_REQUESTED** [review]
Line 22 - change width to 100% to remove the horizontal scroll bar

**Review by Georgeoak03 — COMMENTED** [review]
Comments integrated

**Pull request closed without merging**